test/ffi/base_utils.jl

```diff
@@ -10,7 +10,7 @@ using Test
 
     @test b != bb
     @test b !== bb
-    @test LightGBM.LGBM_BoosterSaveModelToString(b, 0, 0) == LightGBM.LGBM_BoosterSaveModelToString(bb, 0, 0)
+    @test LightGBM.LGBM_BoosterSaveModelToString(b, 0, 0, 0) == LightGBM.LGBM_BoosterSaveModelToString(bb, 0, 0, 0)
```

Comment:
It might have been worth multiple dispatching these methods so you don't have to update every occurrence in the codebase

Reply:
isn't it better with defaults in this case as we showed in tests it doesn't matter which one is used so either the user specifies a value, or we go with a default